modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -3795,7 +3795,7 @@ private CompletableFuture<Boolean> onIndexBuilding(CatalogEventParameters parame
             assert indexDescriptor != null : "indexId=" + indexId + ", catalogVersion=" + parameters.catalogVersion();
 
             if (indexDescriptor.tableId() == tableId()) {
-                txRwOperationTracker.updateMinAllowedCatalogVersionForStartOperation(indexDescriptor.creationCatalogVersion());
+                txRwOperationTracker.updateMinAllowedCatalogVersionForStartOperation(indexDescriptor.txWaitCatalogVersion());
```

Comment:
@rpuch Do you think it is necessary to update at `INDEX_REMOVED`? I think yes.

Reply:
Even if some operation from a dead coordinator executes, it's not a problem: the worst thing it can do is add a tuple to an index that is not writable anymore, it's not a big deal. We can ignore this for the sake of code simplicity.

Also, if we raise minAllowedVersion here, we won't allow to execute an operation from a dead coordinator that we could allow to execute.

Reply:
We discussed it in person, there is no problem here.